apps/arena/lib/arena/game_updater.ex

```diff
@@ -138,6 +139,9 @@ defmodule Arena.GameUpdater do
       |> handle_destroyed_crates(state.game_config)
       |> Effect.apply_effect_mechanic()
       |> Map.put(:server_timestamp, now)
+      # Traps
+      |> activate_traps()
```

Comment:
Maybe this could renamed to `activate_steped_traps` or something like that since this leads me to think that you will activate every trap

Reply:
I agree!